lib/app/components/copy/copy_builder.dart

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: ice License 1.0
+
+//ignore_for_file: avoid_positional_boolean_parameters
```

Comment:
Don't see any related code for this rule

Reply:
removed